src/js/controller/RacingGameController.js

```diff
@@ -68,9 +63,7 @@ export default class RacingGameController {
     }
 
     this.getCountInput();
-    this.racingGame.count > 0 && this.racingGame.runGame();
-    this.view.renderProgress(this.racingGame.getCars());
-    this.view.renderResult(this.racingGame.getWinners());
+    this.racingGame.count > 0 && this.runGame();
```

Comment:
이렇게는 어떨까요? 작성해주신 것도 동작은 하지만 이 자체는 boolean이 됩니다!
```suggestion
    if (this.racingGame.count > 0) {
      this.runGame();
    }
```

Reply:
> 위의 방법에서는 boolean이 되면 문제가 생길 수 있는걸까요?

아니요 ㅎㅎ 동작은 하지만 의미에 맞게 작성하는게 중요하다고 생각해서 제안드려봤습니다 ㅎㅎ